language.md

```diff
@@ -396,8 +396,17 @@ Functions may use constants or previously defined functions (either built-in or
 defined in the file). Functions cannot be defined recursively. The return type
 is determined automatically.
 
+## Constants
+At the top level of a file, constants may be defined:
```

Comment:
`At the top level of a file, ...` -> `At the top level of a file, after any TypeAliases, ...`